.github/workflows/commitlint.yml

```diff
@@ -0,0 +1,12 @@
+name: commit-lint
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: ahmadnassri/action-commit-lint@v2
+        with:
+          config: conventional
```

Comment:
https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#type-enum
It seems that there is no `hotfix` item. I am not sure if `hotfix` is still in use now.

Reply:
I have not seen that distinction being made in the recent past at least. I am very much in favor of simplifying and automating things.